frame/nomination-pools/src/lib.rs

```diff
@@ -2391,15 +2799,20 @@ impl<T: Config> Pallet<T> {
 		member.last_recorded_reward_counter = current_reward_counter;
 		reward_pool.register_claimed_reward(pending_rewards);
 
-		// Transfer payout to the member.
-		T::Currency::transfer(
-			&bonded_pool.reward_account(),
-			&member_account,
-			pending_rewards,
-			// defensive: the depositor has put existential deposit into the pool and it stays
-			// untouched, reward account shall not die.
-			ExistenceRequirement::AllowDeath,
-		)?;
+		// Transfer remaining payout to the member.
```

Comment:
would be good to have a test for when commission is 100%

Reply:
Implemented.